draft-iab-path-signals-collaboration.md

```diff
@@ -400,8 +400,9 @@ very targeted, minimal information carriage mechanisms.
 Some simple forms of information often exist in cleartext
 form, e.g, ECN bits from routers are generally not authenticated
 or integrity protected. This is possible when the information
-exchanges are advisory in their nature, and do not carry any
-significantly sensitive information from the parties.
+exchanges do not carry any significantly sensitive information
+from the parties. Often these kind of interations are also advisory
+in their nature (see also next section).
```

Comment:
Can we add a `{ref}` reference here?

Reply:
You mean a reference to the next section? Sure.

Reply:
Just use {{impact}} for the reference, and then to {#impact} after the title of the next section (limiting impact of information). Mirja, will you make these changes so that we can merge the PR?